src/VariableBorrowInterestRateStrategy.sol

```diff
@@ -1,53 +1,65 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.10;
 
-import { IERC20 }                       from 'aave-v3-core/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
-import { WadRayMath }                   from 'aave-v3-core/contracts/protocol/libraries/math/WadRayMath.sol';
-import { PercentageMath }               from 'aave-v3-core/contracts/protocol/libraries/math/PercentageMath.sol';
-import { DataTypes }                    from 'aave-v3-core/contracts/protocol/libraries/types/DataTypes.sol';
-import { Errors }                       from 'aave-v3-core/contracts/protocol/libraries/helpers/Errors.sol';
+import { IERC20 } from 'aave-v3-core/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+
 import { IDefaultInterestRateStrategy } from 'aave-v3-core/contracts/interfaces/IDefaultInterestRateStrategy.sol';
-import { IReserveInterestRateStrategy } from 'aave-v3-core/contracts/interfaces/IReserveInterestRateStrategy.sol';
 import { IPoolAddressesProvider }       from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
+import { IReserveInterestRateStrategy } from 'aave-v3-core/contracts/interfaces/IReserveInterestRateStrategy.sol';
+
+import { DataTypes }      from 'aave-v3-core/contracts/protocol/libraries/types/DataTypes.sol';
+import { Errors }         from 'aave-v3-core/contracts/protocol/libraries/helpers/Errors.sol';
+import { PercentageMath } from 'aave-v3-core/contracts/protocol/libraries/math/PercentageMath.sol';
+import { WadRayMath }     from 'aave-v3-core/contracts/protocol/libraries/math/WadRayMath.sol';
 
 /**
- * @title VariableBorrowInterestRateStrategy contract
- * @author Aave
- * @notice Implements the calculation of the interest rates depending on the reserve state
- * @dev The model of interest rate is based on 2 slopes, one before the `OPTIMAL_USAGE_RATIO`
- * point of usage and another from that one to 100%.
- * - An instance of this same contract, can't be used across different Aave markets, due to the caching
- *   of the PoolAddressesProvider
- * - Note: This is a modified version of DefaultReserveInterestRateStrategy with the stable borrow feature disabled.
+ * @title  VariableBorrowInterestRateStrategy contract
+ * @author SparkLend
```

Comment:
Original author is Aave we can't change that, only add additional authors if we want, but it's fine.

Reply:
True updated

Reply:
Fixed